chrome_extension/install.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
What do you think about re-implementing this in Python so it works on both platforms? Or is this not necessary on Windows?